CHANGELOG.md

```diff
@@ -16,7 +16,7 @@ scheme are considered to be bugs.
 
 ### Changed
 
-* Your contribution here.
+* [#521](https://github.com/hashie/hashie/pull/499): Do not convert keys that cannot be represented as symbols to String in Mash initialization - [@carolineartz](https://github.com/carolineartz).
```

Comment:
Quote `String` and `Mash`. 

Put back "your contribution here" for the next person.